app/assets/stylesheets/govuk_publishing_components/components/govspeak/_attachment.scss

```diff
@@ -109,6 +109,23 @@
       .js-hidden {
         display: none;
       }
+
+      // stylelint-disable declaration-no-important
+      @include govuk-media-query($media-type: print) {
```

Comment:
The thumbnail image (`.attachment-thumb`) doesn't appear in print view, but there's still the gap on the left where it would be. Would it be possible to remove this? (I can't see where this is already hidden).

![Screenshot 2024-08-08 at 08 40 58](https://github.com/user-attachments/assets/c02c427e-78d8-4b6f-b7c8-4e5e62129af3)

Reply:
Fixed in latest update.